EKF/ekf.h

```diff
@@ -700,8 +699,16 @@ class Ekf final : public EstimatorInterface
 	// measurement update with a single measurement
 	// returns true if fusion is performed
 	template <size_t ...Idxs>
-	bool measurementUpdate(const Vector24f &K, const SparseVector24f<Idxs...> &H, float innovation)
+	bool measurementUpdate(Vector24f &K, const SparseVector24f<Idxs...> &H, float innovation)
 	{
+		if (_accel_bias_inhibit[0]) {
+			K(13) = 0.0f;
+		} else if (_accel_bias_inhibit[1]) {
+			K(14) = 0.0f;
+		} else if (_accel_bias_inhibit[2]) {
+			K(15) = 0.0f;
+		}
+
```

Comment:
Two axes can be inhibited at the same time
```suggestion
		for (unsigned i = 0; i < 3; i++) {
			if (_accel_bias_inhibit[i]) {
				K(13 + i) = 0.0f;
			}
		}
```

Reply:
Squashed into the original commit.

Reply:
TXS